schemas/app-interface/app-interface-email-1.yml

```
@@ -2,6 +2,8 @@
 "$schema": /metaschema-1.json
 version: '1.0'
 type: object
+title: "app-interface-email"
+description: "Specification for single automated email message. Recipients can be static or dynamic based on service, cluster, namespace, aws account, role, or user relations."
```

Could you please use the multiline yaml formatting? Would be easier to read, and anybody will copy that pattern in other/new schema files ;) 

```suggestion
description: |
  Specification for a single automated email message. 
  Recipients can be static or dynamic based on service, cluster, namespace, aws account, role, or user relations.
```